molgenis-data-file/src/main/java/org/molgenis/data/file/model/FileMeta.java

```diff
@@ -22,6 +22,7 @@ public FileMeta(EntityType entityType) {
   }
 
   public FileMeta(String id, EntityType entityType) {
+    super(entityType);
```

Comment:
Same fix is required for ```public FileMeta(String id, EntityType entityType)```

Reply:
I'm not sure what you mean. That's this method.

Reply:
You are right, my bad.